src/deluge/memory/memory_region.h

```diff
@@ -75,6 +75,8 @@ class MemoryRegion {
 
 	void writeTempHeadersBeforeASteal(uint32_t newStartAddress, uint32_t newSize);
 	void sanityCheck();
+
+public:
```

Comment:
If they need to be public should these be moved to the first public section instead?

Reply:
Agreed, fixed